lib/imapoptions

```diff
@@ -2204,6 +2204,18 @@ If all partitions are over that limit, this feature is not used anymore.
   Deprecated. No longer used.
  */
 
+{ "search_index_skip_domains", NULL, STRING, "3.3.0" }
+/*
+ A space separated list of domains - if set, any users in the listed domains
+ will be skipped when indexing.
+ */
+
+{ "search_index_skip_users", NULL, STRING, "3.3.0" }
```

Comment:
Can you please make these "3.3.1", as that will be the first "release" they'll eventually appear in?  Thanks

Reply:
Oh yeah - thanks.  I realised that as I did them and then forgot to come back to it.